src/services/eventing.js

```diff
@@ -1,8 +1,83 @@
+import gql from 'graphql-tag';
+import { ApolloClient } from 'apollo-client';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { HttpLink } from 'apollo-link-http';
+
 class Eventing {
   constructor(client) {
     this.client = client
   }
 
+  fetchEventLogs(projectId, {status, showName, name, showDate, startDate, endDate}, lastEventID){
+    return new Promise((resolve, reject) => {
+      let uri = `/v1/api/${projectId}/graphql`
+      if (process.env.NODE_ENV !== "production") {
+        uri = "http://localhost:4122" + uri;
+      }
+    const cache = new InMemoryCache({ addTypename: false });
+    const link = new HttpLink({ uri: uri });
+    const graphlqlClient = new ApolloClient({
+      cache: cache,
+      link: link
+    });
+
+    graphlqlClient.query({
+      query: gql`
+        query {
+          event_logs (
+            limit: 10,
+            where: {
+              status: {_in: $status},
+              ${showName ? "rule_name: {_eq: $name}" : ""}
+              ${showDate ? "event_timestamp: {_gte: $startDate, _lte: $endDate}" : ""}
+              ${lastEventID ? "_id: {_gt: $lastEventID}": ""}
+            }
+          ) @mysql {
```

Comment:
The database directive shouldn't be hardcoded. Its value should come from the eventing config of a project in redux.